addons/mail/static/src/components/attachment_card/attachment_card.xml

```diff
@@ -28,34 +28,32 @@
                         </t>
                     </div>
                     <!-- Attachment aside -->
-                    <t t-if="(!attachmentCard.attachmentList.composerViewOwner or attachmentCard.attachment.isEditable)">
```

Comment:
Could you confirm, why is it ok to remove the condition?

Reply:
@seb-odoo yes, it's ok to remove it as it's dead code

Reply:
But why is it dead code?
It seemed naturally to me when I created the task, but now I don't remember.

Reply:
Whenever the attachmentcard will be in the composerview it will also be deletable so this condition will always be true.

If the attachmentcard is not in composerview, which means it is in chatter attachment box or in message view then the condition "!attachmentCard.attachmentList.composerViewOwner" will always be true. So in every situation the condition is going to be true. So that's why there is no need for this condition.

And below for every buttons it is checking their respective conditions.

Reply:
Good catch! That's what happens when we use conditional as guesses for what we should do (here: `attachmentCard.attachmentList.composerViewOwner`).

Everything becomes much simpler with just `deletable` determining presence of deletion button or not.

It's out of scope of PR, but I see that other buttons still have `attachmentCard.attachmentList.composerViewOwner`.
This looks also a relics of past when we were fond of riddles in the code.
Could be removed in future PRs!